compose.yaml

```diff
@@ -51,19 +51,23 @@ services:
   ctakes-covid:
     extends: ctakes-covid-base
     profiles:
+      - chart-review
+      - chart-review-gpu
```

Comment:
I realized that with the previous switch to study-specific profiles, the `chart-review` mode, which has its own needs (ctakes and the negation transformer) didn't have a profile, but that it should - like a study does.

Doc changes were also made lower down